docs/DEVELOPING.md

```diff
@@ -201,7 +201,7 @@ $ make ftest NAME=s3
 
 ## Google Cloud Storage Connector
 
-The [Elastic Google Cloud Storage connector](https://github.com/elastic/connectors-python/blob/main/connectors/sources/google_cloud_storage.py) is provided in the Elastic connectors python framework and can be used via [build a connector](https://www.elastic.co/guide/en/enterprise-search/current/build-connector.html).
+The [Elastic Google Cloud Storage connector](https://github.com/elastic/connectors-python/blob/8.7/connectors/sources/google_cloud_storage.py) is provided in the Elastic connectors python framework and can be used via [build a connector](https://www.elastic.co/guide/en/enterprise-search/current/build-connector.html).
```

Comment:
```suggestion
The [Elastic Google Cloud Storage connector](../connectors/sources/google_cloud_storage.py) is provided in the Elastic connectors python framework and can be used via [build a connector](https://www.elastic.co/guide/en/enterprise-search/current/build-connector.html).
```

Reply:
Since we had a discussion with @danajuratoni we decided to point `Network Drive` and `Amazon S3` to `8.6` branch and now `GCS`, `ABS`, and `Generic Database` are part of 8.7 release we pointing to `8.7` branch

Reply:
@wangch079 ideally the users on the 8.7 branch should not leave it when clicking through documentation and the documentation should be moved to elastic.co in about 1 release after the connector client is finished.
Does your proposed change let the users keep the selected branch when clicking links?

Reply:
>ideally the users on the 8.7 branch should not leave it when clicking through documentation

The markdown now contains links to `8.6` and `8.7` branches. E.g. In `8.7`:

https://github.com/elastic/connectors-python/blob/f41c8a1ec50dd7d749b4b1c333a006697ea4c8cf/docs/DEVELOPING.md?plain=1#L5

>Does your proposed change let the users keep the selected branch when clicking links?

Yes. Making the proposed change will keep the users in the selected branch.

Reply:
If `../connectors/sources/` keeps users in the selected branch, I'm good to use this for all our links!

Reply:
Updated with relative links, and we will raise PR for other connectors soon.